### ISSUE TITLE
BatchNorm beta diminishing parameter values.

### ISSUE THREAD

**Issue by aksrajvanshi** [awslabs/djl#132]
@stu1130 

I am running the BatchNorm section, and I made gamma and beta as the two parameters in the section. Although the results turn out to be fine, the beta is turning out with really low values. 

This is the book's result:

![Screen Shot 2020-07-14 at 2 27 25 PM](https://user-images.githubusercontent.com/16722550/87462931-af23ec00-c5de-11ea-8484-c49fd52d2849.png)

And this is mine:

![Screen Shot 2020-07-14 at 2 26 54 PM](https://user-images.githubusercontent.com/16722550/87462955-b8ad5400-c5de-11ea-97aa-f7d6dcdfb4f6.png)

**Comment by stu1130** [comment]
I can reproduce the issue and found both PyTorch and MXNet have the same problem. This could be caused by gradient vanishing. Only the first batchNorm have the issue. need to dive deeper

**Comment by lanking520** [comment]
Reproducible use general batchnorm implementation. Investigating now

**Comment by goswamig** [comment]
@lanking520  do we have any update?
